### PR TITLE
close camera calibration at end of application

### PR DESCRIPTION
**Pull request by atsju**
mention #82

### CODE REVIEW

**Review by githubdoe — APPROVED** [review]
Just for information.  Camera Calibration was never used by anyone as far as I can tell.  It also was never verified that it worked correctly.  I got confused by different nomenclatures and was not able to compare with an example I found on the net that I had planned to use for verification.  It probably could be removed.  I hated to remove it because it was a lot of work and research that might be useful.  However it probably requires a large learning curve to actual be used by anyone.  I have abandoned all thought of ever using it.  Most camera and lens combinations we ATM's used in the past did not create distorted views enough to matter.

**Comment by gr5** [comment]
I tried to use it but it wouldn't recognize the pattern.  Let me clarify:
Step 1: create pattern.  Works great.
Step 2: photograph pattern with camera and lens.  Works great.
Step 3: import photograph and calculate distortion of photographed pattern.  Could not get it to work.

As part of step 2, supposedly you should also use a specific "stop" device but I never got that far.

**Comment by githubdoe** [comment]
I remember that could happen sometimes.  But no longer remember the details.  There were/are HTML links in the code comments or help file I think pointing to where I got the algorithms and web sites.  IIRC.

**Comment by githubdoe** [comment]
I also remember that I was planning on validating it against another app where one could add distortion to a photo.  I would add the distortion to the grid pattern.  However the units and names it called the distortion did not match with any of the names or units of the calibration constants my app produced.  So I could not compare them directly and did not know how to reconcile that issue.  Thus there was no validation of my algorithm.

File: mainwindow.cpp [hunk 1]
```diff
@@ -282,7 +281,7 @@ void MainWindow::openWaveFrontonInit(QStringList args){
         if (pd.wasCanceled())
             break;
 
-        if (arg.toUpper().endsWith(".WFT")){
+        if (arg.endsWith(".wft", Qt::CaseInsensitive)){
```
Comment (gr5) [inline mainwindow.cpp:284]: I'd really rather you don't change things like this.  The code is about equally easy to read either way.  The code does "the same thing" either way.  There is a tiny (very tiny) chance you will add a bug anytime you change *anything*.  And we don't need to worry about efficiencies.  If this code looped 10,000 times, maybe?  Certainly if it looped a billion times.  Some people might find the old code easier to read.

Having said all that I'd prefer you to leave it this new way as I plan to build and test this in the next few minutes and merge it.

Comment (gr5) [reply mainwindow.cpp:284]: Or was there a clangd/warning about the old line of code?

**Comment by githubdoe** [comment]
That is a change that I like and would have done myself at the time had I knew about that option.

**Comment by githubdoe** [comment]
However it does not make a difference for .wft files since this is the program that creates the extension.

**Pull request merged**